pkg/resources/openstack/instances.go

```diff
@@ -29,32 +29,31 @@ const (
 
 func (os *clusterDiscoveryOS) ListInstances() ([]*resources.Resource, error) {
 	var resourceTrackers []*resources.Resource
-	opt := servers.ListOpts{
-		Name: os.clusterName + "?",
-	}
-	instances, err := os.osCloud.ListInstances(opt)
+	instances, err := os.osCloud.ListInstances(servers.ListOpts{})
 	if err != nil {
 		return resourceTrackers, err
 	}
 
 	for _, instance := range instances {
-
-		// Clean up any bound floating IP's
-		floatingIPs, err := os.listFloatingIPs(instance.ID)
-		if err != nil {
-			return resourceTrackers, err
-		}
-		resourceTrackers = append(resourceTrackers, floatingIPs...)
-
-		resourceTracker := &resources.Resource{
-			Name: instance.Name,
-			ID:   instance.ID,
-			Type: typeInstance,
-			Deleter: func(cloud fi.Cloud, r *resources.Resource) error {
-				return cloud.(openstack.OpenstackCloud).DeleteInstanceWithID(r.ID)
-			},
+		val, ok := instance.Metadata["k8s"]
```

Comment:
FYI on AWS we use `kubernetes.io/cluster/<cluster-name>=shared` and `kubernetes.io/cluster/<cluster-name>=owned`.  `<cluster-name>` can be anything but for kops we use the name of the cluster you create.

We originally had `KubernetesCluster=<cluster-name>` but this made it hard to have shared resources.

Reply:
(and it looks like GCP is using `kubernetes.io/cluster-id=<cluster-name>`, but only in the description field for GCLBs https://github.com/kubernetes/kubernetes/blob/1c557b9ce866d67ec6088f37058e8594b89606ee/pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go#L862